openarm_hardware/include/openarm_hardware/canbus.hpp

```diff
@@ -10,17 +12,30 @@
 #include <linux/can/raw.h>
 #include <array>
 #include <fcntl.h>
+#include <string>
+//#include "global.h"
+
+#define CAN_MODE_CLASSIC 0
+#define CAN_MODE_FD      1
```

Comment:
`enum class` is better than `int` + `#define` for better compile time check.
For example, compiler can report an error or warning when we use unknown enum value.

FYI: Google C++ Style Guide has a guide for enum names: https://google.github.io/styleguide/cppguide.html#Enumerator_Names